tsl/test/expected/continuous_aggs_invalidation.out

```diff
@@ -1163,3 +1163,23 @@ ORDER BY 1,2;
      10 |      1 |       10
 (1 row)
 
+-- Test that adjacent invalidations are merged
+INSERT INTO conditions VALUES(1, 1, 1.0), (2, 1, 2.0);
```

Comment:
it will be easier to read if it is on separate line as others, otherwise it feels like a hole between `1` and `3`.

Reply:
This is intentional in order to create a range invalidation between 1-2. Each statement is its own transaction and thus records a separate invalidation. I want to test a mix of both non-trivial (length > 1) and trivial (length=1) ranges.

Reply:
Thank you for clarifying!
May be:
```suggestion
INSERT INTO conditions VALUES (1, 1, 1.0), 
                                                         (2, 1, 2.0);
```
It's not important.